spec/services/spree/product_feed_service_spec.rb

```diff
@@ -86,7 +86,7 @@
   describe '#url' do
     subject { service.url }
 
-    it { is_expected.to eq("#{store.url}/#{product.slug}") }
+    it { is_expected.to eq("https://#{store.url}/#{product.slug}") }
```

Comment:
I'd like to see a test that covers the case where the url has the `http` in it too.

Reply:
Good call, added one.